web/pdf_viewer.js

```diff
@@ -2299,6 +2299,9 @@ class PDFViewer {
    * @param {AnnotationEditorModeOptions} options
    */
   set annotationEditorMode({ mode, editId = null, isFromKeyboard = false }) {
+    if (mode === AnnotationEditorType.STAMP) {
+      this.#mlManager?.loadModel("altText");
+    }
```

Comment:
Can we please move this *after* all of the existing checks instead, since it shouldn't be needed if those fail?
Basically, it seems to me that this block should go *just after* this line instead:
https://github.com/mozilla/pdf.js/blob/08821a3e5de64221b69f6391e555bb25c1edae01/web/pdf_viewer.js#L2313

Reply:
Additionally, I believe that we might *also* want to duplicate this block *just before* this line:
https://github.com/mozilla/pdf.js/blob/08821a3e5de64221b69f6391e555bb25c1edae01/web/pdf_viewer.js#L909

Reply:
Sorry, but what about https://github.com/mozilla/pdf.js/pull/18546#discussion_r1701888216?

Obviously that case only applies if the "annotationEditorMode" preference has been set to auto-enable Stamp-editing on PDF document load.

Reply:
Sorry I missed the mentioned comment.